integration-tests/tests/run-catalog-workflow.js

```diff
@@ -10,23 +10,15 @@ const testCatalogFlowFn = _.flow(
 )(async ({ page, testUrl, token, workspaceName }) => {
   await enableDataCatalog(page, testUrl, token)
```

Comment:
The first 6 lines of this test are identical to lines 7-12 from `link-to-new-workspace`. I would recommend that you share the code between the two tests.